src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -46,11 +46,16 @@ public void sendConsentedHWFSuccessfulConfirmationEmail(CaseDetails caseDetails)
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(caseDetails), uri);
     }
 
-    public void sendAssignToJudgeConfirmationEmail(CaseDetails caseDetails) {
+    public void sendAssignToJudgeConfirmationEmailToApplicantSolicitor(CaseDetails caseDetails) {
         URI uri = buildUri(notificationServiceConfiguration.getAssignToJudge());
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(caseDetails), uri);
     }
 
+    public void sendAssignToJudgeConfirmationEmailToRespondentSolicitor(CaseDetails caseDetails) {
+        URI uri = buildUri(notificationServiceConfiguration.getAssignToJudge());
```

Comment:
see how we've done this for other methods, we have a base method that we call fro for app and resp specific methods

Reply:
done 👍